File: app/src/main/java/org/imozerov/vkgroupdialogs/util/DateUtil.java
package org.imozerov.vkgroupdialogs.util;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;


public class DateUtil {
    public static String chatLastMessage(Date date) {
        // TODO remove this! date in valid data should not be null!
        if (date == null) {
            return "";
        }
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("HH:mm", Locale.ENGLISH);
        String formattedDate = simpleDateFormat.format(date);
        return formattedDate;
    }
}

File: app/src/main/java/org/imozerov/vkgroupdialogs/util/DateUtil.kt
package org.imozerov.vkgroupdialogs.util

import java.text.SimpleDateFormat
import java.util.*


object DateUtil {
    private val timeZone = TimeZone.getDefault()

    fun chatLastMessage(date: Long): String {
        val simpleDateFormat = SimpleDateFormat("HH:mm", Locale.ENGLISH)
        simpleDateFormat.timeZone = timeZone
        val formattedDate = simpleDateFormat.format(date)
        return formattedDate
    }
}
